scte35/segmentationdescriptor.go

```diff
@@ -229,6 +253,16 @@ func (d *segmentationDescriptor) UPID() []byte {
 	return d.upid
 }
 
+func (d *segmentationDescriptor) StreamSwitchSignalId() string {
+	var signalId string
+	// SignalId is present in the MID.
+	if len(d.mid) == 2 {
```

Comment:
We probably shouldn't rely on the structure of stream switch signals being consistent.  We should iterate over d.mid, testing each UPID to see if it's a switch signal.